### ISSUE TITLE
Unable to debug (F5) new blazorwasm project in Visual Studio using .NET 6.0 Preview 3

### ISSUE THREAD

**Issue by adrianwright109** [dotnet/aspnetcore#31798]
**Issue**
I came across this issue when trying to repro a different issue I am facing #31799.

This issue stops me from debugging a new `blazorwasm` project created from the `CLI` using `F5` in Visual Studio 2019 version `16.10.0 Preview 1.0`

**To Reproduce**
1. `md BreakPointsStayActive`
2. `cd .\BreakPointsStayActive\`
3. `dotnet new blazorwasm`
4. Open project `.\BreakPointsStayActive\BreakPointsStayActive.csproj` in Visual Studio 2019 version `16.10.0 Preview 1.0`
5. Set a breakpoint on line 14 (`currentCount++;`) of the `Counter.razor` page
6. Press `F5` to launch the debugger in Chrome

**Expected**
A new chrome window is spun up with the application URL and I'm able to hit the breakpoint when incrementing the counter.

**Actual**
A new chrome windows is spun up but it remains on `about:blank`
![image](https://user-images.githubusercontent.com/47532258/114698701-4f967080-9d17-11eb-85ec-4408d2904cb0.png)

An error then popups up in Visual Studio 2019
![image](https://user-images.githubusercontent.com/47532258/114698789-663cc780-9d17-11eb-9dc7-dd2d36c3565c.png)

**Further technical details**
- ASP.NET Core version: `6.0.100-preview.3.21202.5`
- Include the output of `dotnet --info`
```
.NET SDK (reflecting any global.json):
 Version:   6.0.100-preview.3.21202.5
 Commit:    aee38a6dd4

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19042
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\6.0.100-preview.3.21202.5\

Host (useful for support):
  Version: 6.0.0-preview.3.21201.4
  Commit:  236cb21e3c

.NET SDKs installed:
  3.1.300-preview-015048 [C:\Program Files\dotnet\sdk]
  3.1.408 [C:\Program Files\dotnet\sdk]
  5.0.100-rc.1.20452.10 [C:\Program Files\dotnet\sdk]
  5.0.104 [C:\Program Files\dotnet\sdk]
  5.0.200-preview.20614.14 [C:\Program Files\dotnet\sdk]
  5.0.200-preview.21077.7 [C:\Program Files\dotnet\sdk]
  5.0.202 [C:\Program Files\dotnet\sdk]
  6.0.100-preview.3.21202.5 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.All 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.23 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.27 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.23 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.27 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.0.0-preview7.19365.7 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.9 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.14 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.1 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.2 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.4 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.5 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.0-preview.3.21201.13 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.23 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.27 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.0.0-preview7-27912-14 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.9 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.14 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.1 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.2 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.4 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.5 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.0-preview.3.21201.4 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.0.0-preview7-27912-14 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 3.1.9 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 3.1.14 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.1 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.2 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.4 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.5 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.0-preview.3.21201.3 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```
- Visual Studio 2019 version `16.10.0 Preview 1.0`

**Comment by adrianwright109** [comment]
I have also reported this issue via Visual Studio so you should have access to log files:
https://developercommunity.visualstudio.com/t/Unable-to-debug-F5-new-blazorwasm-proj/1398111

**Comment by adrianwright109** [comment]
**Update**
The same thing happens if I create the new `blazorwasm` project (repro steps `1-3` above) solely using Visual Studio 2019

**Comment by captainsafia** [comment]
@adrianwright109 Did you happen to be using an IIS Express server to debug? There's some known issues with launching Blazor WASM app with IIS Express that we resolved in 6.0.0-preview4.

Using Kestrel to debug should be a sufficient workaround for now. If you run into this when using Kestrel, then something might be afoot.

**Comment by adrianwright109** [comment]
@captainsafia - yes it was using IIS Express as that is the default in the template.  I will switch it to Kestrel to see if that works. It's not a big issue for me as I only ran into the issue when trying to create a repro for #31799 which I managed to do not using .NET 6.0 previews.
